pkg/common/scheduler/README.md

```diff
@@ -0,0 +1,247 @@
+# Knative Eventing Multi-Tenant Scheduler
+
+[![LICENSE](https://img.shields.io/github/license/knative-sandbox/eventing-kafka.svg)](https://github.com/knative-sandbox/eventing-kafka/blob/master/LICENSE)
+[![Slack](https://img.shields.io/badge/Signup-Knative_Slack-white.svg?logo=slack)](https://slack.knative.dev)
+[![Slack](https://img.shields.io/badge/%23eventing-white.svg?logo=slack&color=522a5e)](https://knative.slack.com/archives/C9JP909F0)
+
+## Scheduler with High-Availability
```

Comment:
Before this section, what about defining some of the new concepts? I'm thinking at the minimum `vreplicas` (short for virtual replicas) and vpod (short for virtual pod).

Reply:
Added.